linera-views/src/dynamo_db.rs

```diff
@@ -568,14 +653,28 @@ impl DynamoDbClientInternal {
             return Ok(true);
         };
         let test = match &error {
-            SdkError::ServiceError { err, raw: _ } => match &err.kind {
-                GetItemErrorKind::ResourceNotFoundException(inner_error) => {
-                    inner_error.message
+            SdkError::ServiceError(error) => match error.err() {
+                GetItemError::ResourceNotFoundException(error) => {
+                    println!("2 : error={:?}", error);
+                    if error.message
                         == Some("Cannot do operations on a non-existent table".to_string())
+                    {
+                        println!("Exit case 1");
```

Comment:
What is this?

Reply:
Some oversight.

Reply:
Even in tests, log messages need to be meaningful and use the `tracing` crate (and not `print!`).
Can't we log the error types returned by DynamoDb?

Reply:
Alright, it looks like this is removed in later commits.